qiita_db/processing_job.py

```diff
@@ -7,10 +7,62 @@
 # -----------------------------------------------------------------------------
 
 from uuid import UUID
-
+from datetime import datetime
+from subprocess import Popen, PIPE
+from multiprocessing import Process
+from os.path import join
+from itertools import chain
+from collections import defaultdict
+from json import dumps
+
+from future.utils import viewitems, viewvalues
+import networkx as nx
+
+from qiita_core.qiita_settings import qiita_config
 import qiita_db as qdb
 
 
+def _system_call(cmd):
+    """Call cmd and return (stdout, stderr, return_value)
+
+    Parameters
+    ----------
+    cmd : str or iterator of str
```

Comment:
Should this have a test for an iterator of str? I can see one for success (str) and error.